components/styled/FeatureBackground.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+
+const FeatureBackground = styled.div`
+  width: ${props => (props.welcome || props.login ? "100%" : "98%")};
+  height: ${props =>
+    props.welcome ||
+    /* props.transactions || */
```

Comment:
Please delete code you don't use anymore.

Reply:
done!